plugins/riemann-controller/riemann_controller/resources/deployment.config.template

```diff
@@ -14,6 +15,12 @@
         (processing-queue "{{ctx.deployment.id}}" {:queue-size     1000
                                                    :core-pool-size 1
                                                    :max-pool-size  4})]
+(def node-instances-stream {
```

Comment:
make it optional - sometimes we want to define policy for a group of nodes. two options:
1. node instances
2. all that are part of the members section